src/snowflake/snowpark/modin/plugin/_internal/indexing_utils.py

```diff
@@ -258,7 +258,9 @@ def convert_positional_key(
     integer: the Cython equivalent of `isinstance(val, (int, long, np.integer)) and not isinstance(val, bool)`.
             See source code in pandas/_libs/tslibs/util.pxd:is_integer_object for details.
     """
-    from snowflake.snowpark.modin.pandas.indexing import is_range_like
+    from snowflake.snowpark.modin.plugin.extensions.indexing_overrides import (  # pragma: no cover
```

Comment:
how come the #pragma: no cover is necessary?

Reply:
This line caused the coverage check to fail.

Reply:
+1

Reply:
why the code coverage didn't run this line?

Reply:
Actually, it seems this whole function (`convert_positional_key `)is not used anywhere. The code coverage check only focuses on the newly introduced changes. And since this line is the only change made by this PR for this function, it was flagged.

I can have a separate PR to remove this function being dead code. Wdyt?

Reply:
sgtm! You can remove here too.

Reply:
Agreed, I'm fine with bundling the removal in this PR.